src/clients/python/library/tritonclient/utils/__init__.py

```diff
@@ -122,7 +122,7 @@ def np_to_triton_dtype(np_dtype):
         return "FP32"
     elif np_dtype == np.float64:
         return "FP64"
-    elif np_dtype == np.object or np_dtype.type == np.bytes_:
+    elif np_dtype == np.object_ or np_dtype.type == np.bytes_:
```

Comment:
Does this break backwards compat?

Reply:
I don't think it breaks the backward compat. I'll add a test for this in `L0_string_io`. The reason that I changed this in every location is that `np.object` is deprecated and should be replaced with `np.object_`. It looks like it is only a convention that `numpy` is trying to follow. All the types that their equivalent type exists in Python are superseded with an underscore.